src/main/java/com/library/service/AuthorService.java

```diff
@@ -0,0 +1,19 @@
+package com.library.service;
+
+
+import com.library.model.dto.AuthorDto;
+
+import java.util.List;
+
+
+public interface AuthorService {
+
+    List<AuthorDto> findAllWithPageable(Integer page, Integer sizeOnPage);
+
+    AuthorDto findById(Long id);
+
+    AuthorDto save(AuthorDto authorDto);
+    AuthorDto update(AuthorDto authorDto);
```

Comment:
Single style: or empty line separated or without separation

Reply:
added separate line in other part too